src/main/kotlin/uk/gov/dluhc/printapi/config/PdfConfiguration.kt

```diff
@@ -17,7 +17,10 @@ class PdfConfiguration {
     fun temporaryCertificateExplainerPdfTemplateDetailsFactory(
         explainerPdfTemplateProperties: TemporaryCertificateExplainerPdfTemplateProperties,
     ): ExplainerPdfTemplateDetailsFactory =
-        ExplainerPdfTemplateDetailsFactory(explainerPdfTemplateProperties)
+        ExplainerPdfTemplateDetailsFactory(explainerPdfTemplateProperties) {
+            eroId: String, gssCode: String ->
+            "Temporary certificate explainer document not found for eroId $eroId and gssCode $gssCode"
+        }
```

Comment:
This might be easier to see in the IDE, but `ExplainerPdfTemplateDetailsFactory` now has a constructor argument which is a function to generate the exception message. 
Passing high-order functions and lambdas as arguments! Check me out! 😁